.circleci/config.yml

```
@@ -0,0 +1,29 @@
+version: 2.1
+jobs:
+  release-charts:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - run:
+          name: checkout
+          command: |
+            git clone https://${GITHUB_TOKEN}@github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git .
+      - run:
+          name: release
+          command: |
+            echo "export CR_REPO_URL=https://falcosecurity.github.io/charts" >> $BASH_ENV
+            echo "export GIT_USERNAME=$CIRCLE_PROJECT_USERNAME" >> $BASH_ENV
+            echo "export GIT_REPOSITORY_NAME=$CIRCLE_PROJECT_REPONAME" >> $BASH_ENV
+            .circleci/install_tools.sh
+            .circleci/release.sh
+
+workflows:
+  version: 2
+  release:
+    jobs:
+      - release-charts:
```

```suggestion
      - release-charts:
          context: falco
```
Add `context` to inject `GITHUB_TOKEN`